orte/mca/grpcomm/direct/grpcomm_direct_component.c

```diff
@@ -55,7 +55,7 @@ static int direct_register(void)
     /* make the priority adjustable so users can select
      * direct for use by apps without affecting daemons
      */
-    my_priority = 1;
+    my_priority = 100;
```

Comment:
wellll...as we both know, that isn't technically true. we set the priority to much higher values in a number of places. i picked this one because rcd is at 80, but now that we have opal_ignore'd it, i can turn down the heat